bims/views/non_validated_user_list.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+from braces.views import LoginRequiredMixin
+from django.views.generic import ListView
+from bims.models.biological_collection_record import BiologicalCollectionRecord
+
+
+class NonValidatedObjectsUserView(LoginRequiredMixin, ListView):
+
+    model = BiologicalCollectionRecord
+    context_object_name = 'biorecords'
+    template_name = 'non_validated_user_list.html'
+    paginate_by = 10
+
+    def get_context_data(self, **kwargs):
+        filter_name = self.request.GET.get('original_species_name', None)
+        filter_date_to = self.request.GET.get('date_to', None)
+        filter_date_from = self.request.GET.get('date_from', None)
+        context = super(
+            NonValidatedObjectsUserView, self).get_context_data(**kwargs)
+        context['custom_url'] = ''
+        if filter_name:
+            context['custom_url'] = \
+                '&original_species_name={}'.format(filter_name)
+        if filter_date_from:
+            context['custom_url'] = \
```

Comment:
no append text in here? "+="

Reply:
no @meomancer , because `filter_date_from` is only together with `filter_date_to` so only `filter_date_to` needs append text

Reply:
maybe you need to update docs of the class

Reply:
will do